packages/features/shell/Shell.tsx

```diff
@@ -995,11 +995,11 @@ export function ShellMain(props: LayoutProps) {
     <>
       {(props.heading || !!props.backPath) && (
         <div
-          className={classNames(
+          className={`${classNames(
             "flex items-center md:mb-6 md:mt-0",
             props.smallHeading ? "lg:mb-7" : "lg:mb-8",
             props.hideHeadingOnMobile ? "mb-0" : "mb-6"
-          )}>
+          )} px-2`}>
```

Comment:
no need to add px-2 here if you cann add it in the first row next to flex items-center etc.